docs/en/05-supported-backends/generate_backends_md_table.md

```diff
@@ -0,0 +1,61 @@
+# How to generate supported-backends markdown table
+
+This tutorial describes how to generate supported-backends markdown table.
+
+## 1.python Environment
+
+```
+pip install easydict
+```
+
+## 2.Usage
```

Comment:
usage and description should change the format  .could refer to https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/useful_tools.md#torch2onnx